app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,27 @@
 
     {%- endif %}
 
+    <script>
+        {
+            "@context": "https://schema.org",
+            "@type": "ScholarlyArticle",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
```

Comment:
Suspect we might want to use the DOI here.

Reply:
Fountain have advised us to use the URL on this one